bin/options.js

```diff
@@ -69,9 +70,26 @@ function getOptions() {
     return;
   }
 
+  try {
+    const testDir = JSON.parse(
+      fs.readFileSync(path.join(process.cwd(), 'package.json'), 'utf8')
+    ).directories['test'];
+
```

Comment:
This finds the user's "package.json", regardless of where Mocha was installed?
Would think you would need to use "read-pkg-up" npm package.
For example, see [here](https://github.com/yargs/yargs/blob/497194d0cf4253182aa32eed6cadfe6bc8507413/yargs.js#L358-L372)

Reply:
I will have a look at 'read-pkg-up' tomorrow. What do you think of this solution?
`const testDir = require('package.json').directories.test;`
Are there several package.json files? For node and for mocha, globally/locally installed?

Reply:
ok, done.

Reply:
I mentioned in the chat room that I'd prefer using [findup-sync](https://npm.im/findup-sync), since I need to use that in #3556.

Reply:
@boneskull, used this since "yargs" does as well and provides us with a debugged, ready-to-go recipe (see example link)? Since they _will_ be a dependency, "read-pkg-up" package adds no extra cost...

Reply:
@plroebuck No, user's "package.json" can't be found in any situation, that's not realistic. I added above some "Additional information".